PowerPointLabs/PowerPointLabs/Ribbon1.cs

```diff
@@ -1,2300 +1,1259 @@
-        {
-            return TextCollection.AddZoomOutButtonSupertip;
-        }
-        public string GetZoomToAreaButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.ZoomToAreaButtonSupertip;
-        }
-        
-        public string GetAddSpotlightButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.AddSpotlightButtonSupertip;
-        }
-
-        public string GetSpotlightPropertiesButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.SpotlightPropertiesButtonSupertip;
-        }
-        
-        public string GetGenerateRecordButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.GenerateRecordButtonSupertip;
-        }
-        public string GetAddRecordButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.AddRecordButtonSupertip;
-        }
-        public string GetRemoveAudioButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.RemoveAudioButtonSupertip;
-        }
-
-        public string GetAddCaptionsButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.AddCaptionsButtonSupertip;
-        }
-        public string GetRemoveCaptionsButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.RemoveCaptionsButtonSupertip;
-        }
-        public string GetRemoveAllNotesButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.RemoveAllNotesButtonSupertip;
-        }
-
-        public string GetHighlightBulletsTextButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.HighlightBulletsTextButtonSupertip;
-        }
-        public string GetHighlightBulletsBackgroundButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.HighlightBulletsBackgroundButtonSupertip;
-        }
-
-        public string GetHighlightTextFragmentsButtonSupertip(Office.IRibbonControl control)
-        {
-            return TextCollection.HighlightTextFragmentsButtonSupertip;
+        public void AnimationPropertiesEdited(float newDuration, bool newFrameChecked)
+        {
+            try
+            {
+                DefaultDuration = newDuration;
+                FrameAnimationChecked = newFrameChecked;
+                AnimateInSlide.defaultDuration = newDuration;
+                AnimateInSlide.frameAnimationChecked = newFrameChecked;
+                AutoAnimate.defaultDuration = newDuration;
+                AutoAnimate.frameAnimationChecked = newFrameChecked;
+            }
+            catch (Exception e)
+            {
+                Logger.LogException(e, "AnimationPropertiesEdited");
+                throw;
+            }
+        }
+
+        public void ZoomLabDialogButtonPressed(Office.IRibbonControl control)
```

Comment:
This can't be changed yet. We probably need a lot more work to change this, and I think it should be done in a separate PR